src/common/latest/helpers/__tests__/azdo-api-utils-test.ts

```diff
@@ -15,7 +15,7 @@ it("should build jsonpath body properly", async () => {
   jest.spyOn(azdoApiUtils, "getWebApi").mockImplementation(() => webApi);
   jest.spyOn(tl, "debug").mockImplementation(() => null);
 
-  const jsonAsString = `[{\"op\":${Operation.Add},\"path\":\"/sonarglobalqualitygate\",\"value\":\"test\"}]`;
+  const jsonAsString = `[{"op":${Operation.Add},"path":"/sonarglobalqualitygate","value":"test"}]`;
```

Comment:
why did the path formatting need to change?

Reply:
This fixes 8 SonarQube findings